### DISCUSSION TITLE
show pcap for this event is missing

### DISCUSSION THREAD

**Discussion by Alexie30** [Security-Onion-Solutions/securityonion#9718] [Unsupported Versions]
I do the training on youtube and in the video Security Onion Essentials - Ad Hoc Hunting, in hunt, show pcap for this event is missing and I can't find the same port and especially port 8008 to do the steps as in the video, I do not understand where is the problem because for the other videos, I had no problem, thank you for your help

**Comment by InfosecGoon** [comment]
Are you importing a PCAP to work with? What sort of event are you trying to pivot to PCAP from?
